bsp/ls2kdev/applications/pmcon.c

```diff
@@ -0,0 +1,15 @@
+#include <rtthread.h>
+
+static void reboot()
+{
+	rt_hw_cpu_reset();
```

Comment:
Please take care of indentation.

Reply:
ok , I'll fix it. and ask a question please, how to avoid it , i am eidt code with vim and set ts=4. thank you